### ISSUE TITLE
Tree shaking is not working properly with Vite and React (healthicons-react)

### ISSUE THREAD

**Issue by hcondori** [stnrd/healthicons#26]
In a React Vite project using healthicons-react, even using one icon causes the bundler to include the whole healthicons library for the production build.

**Comment by stnrd** [comment]
What if you use `healthicons-react/filled` for example does that help. I am not a vite user myself soo I have to dive a bit into vite to see the issue and find the solution.

**Comment by hcondori** [comment]
Yes, it helps, but only to some extent. In a blank Vite React project (created with `npm create vite@latest`) that imports only one icon, importing from "healthicons-react" causes the importing JS asset to be over 4.4MB, and importing from "healthicons-react/filled" reduces the size to about 1.9MB.

**Comment by stnrd** [comment]
I had some time to look into this and I think I have solved the tree shaking issue. I have added `"sideEffects": false` to all package.json files.

I have tested it by creating an empty React app with vite and ran the build **with** and **without** icons imported. The actual build with icons is ~154kb and without around ~151kb. Soo it looks like the tree shaking is working. 

It's fixed in the release [3.2.1](https://github.com/stnrd/healthicons/releases/tag/v3.2.1)

**Comment by hcondori** [comment]
It's working for me, too. Thank you!
